docs/output_fields.rst

```diff
@@ -435,3 +435,81 @@ Borehole samples (grondmonsters)
     fractie,10,float,0
     methode,10,string,ZEEFPROEF
 
+Bodemsites
+--------------------------------
+  .. csv-table:: Bodemsites
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemsite,1,string,https://www.dov.vlaanderen.be/data/bodemsite/2013-000180
+	naam,1,string,Meise_Neerpoorten
+	datum,1,date,nan
+	beschrijving,1,string,grasland
+	waarnemingsdatum,10,date,nan
+	invoerdatum,10,date,nan
+
+
+BodemLocaties
+--------------------------------
+  .. csv-table:: Bodemlocaties
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemlocatie,1,string,https://www.dov.vlaanderen.be/data/bodemlocatie/2011-000002
+	naam,1,string,STARC_4
+	type,1,string,profielput
+	datum,1,date,nan
+	doel,1,string,archeologische landschappelijke profielputten
+	x,1,float,206553.85
+	y,1,float,168891.11
+	z,1,float,44.00
+	erfgoed,1,boolean,true
+	bodemstreek,1,string,Zandleemstreek
+	waarnemingsdatum,10,date,nan
+	invoerdatum,10,date,nan
+	educatieve_waarde,10,string,ZEER
+
+
+Bodemmonsters
+--------------------------------
+  .. csv-table:: Bodemmonsters
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemmonster,1,string,https://www.dov.vlaanderen.be/data/bodemmonster/1964-264869
+	identificatie,1,string,KART_PROF_073E/12_H2_M1
+	datum_monstername,1,date,1964-11-12
+	tijdstip_monstername,10,time,NaN
+	type,1,string,ENK
+	monsternamedoor,1,string,Centrum voor Grondonderzoek (C.V.G.)
+	techniek,1,string,NaN
+	condities,1,string,Zie scan analoge profielbeschrijving
+	van,1,float,30
+	tot,1,float,45
+	labo,1,string,Centrum voor Grondonderzoek (C.V.G.)
+
+
+Bodemobservaties
+--------------------------------
+  .. csv-table:: Bodemobservaties
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemobservatie,1,string,https://www.dov.vlaanderen.be/data/bodemobservatie/2019-349078
+	pkey_bodemlocatie,1,string,https://www.dov.vlaanderen.be/data/bodemlocatie/1952-007078
+	Aan,1,string,bodemlocatie/1952-007078
+	pkey_parent,1,string,https://www.dov.vlaanderen.be/data/bodemlocatie/1952-007078
+	parameter,1,string,organische_c_perc
+	parametergroep,10,string
+	waarde,1,string,0.38
+	eenheid,1,string,%
+	ondergrens,10,float,NaN
+	bovengrens,10,float,NaN
```

Comment:
We do not include `ondergrens` and `bovengrens` in the Grondwatermonsters type. And we do include `detectie` there.

Can we align the fields in both types?

Reply:
Ondergrens, bovengrens and detectie are not used in bodem. So these can be removed from this type.

Reply:
Why did you add them if they are not used? :smile: 

If you're sure `detectie` is never used (why not?) we can skip it here, but otherwise I'd prefer to align it with Grondwatermonsters.

Reply:
Voor de koolstofmonitoring hebben we net wel de eerste data binnengekregen met een detectielimiet (namelijk bij anorganische koolstof). Dus hebben we vanaf nu wel die detectielimiet nodig. Ik documenteer dit op confluence en kijk welke issues in jira ik hier voor kan loggen. We hebben dit ooit mee in de analyse genomen omdat het bij water zo ontwikkeld was maar niet verder uitgewerkt omdat we tot nu toe er nog geen behoefte voor hadden bij bodem.
Bij pH is er eigenlijk wel een ondergrens en bovengrens dus dit kan ik invullen bij de parameters rond pH.

Reply:
In that case I would align it already with the Grondwatermonster type in pydov and add `detectie` here too. It is valid accoring to the XSD schema, so should work once there is data available in the XML export.

Reply:
I see that you added `ondergrens` and `bovengrens` both here as well as for Grondwatermonsters. If that is your evaluation that's fine for me, but I understood that these were not used in bodem (and not currently available in Grondwatermonsters in pydov), so I would skip these and just add `detectie` to Bodemobservaties.

Reply:
After re-read the comment of Katrien Oorts, I removed "ondergrens" and "bovengrens" from the documentation and bodemobservatietype.

Reply:
Can you remove the added `ondergrens` and `bovengrens` from the Grondwatermonsters type?

I would also love it if the order of the relevant fields in Bodemobservaties would be the same as it is in Grondwatermonsters, i.e.: parametergroep, parameter, detectie, waarde, eenheid